hedera-mirror-rest-java/src/main/java/com/hedera/mirror/restjava/service/NftAllowanceRequest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.mirror.restjava.service;
+
+import lombok.Builder;
+import lombok.Data;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.data.domain.Sort;
+
+@Data
+@Builder
+public class NftAllowanceRequest {
+
+    @NotNull
+    private Long accountId;
+
+    @Builder.Default
+    private Integer limit = 25;
+
+    @Builder.Default
+    private Sort.Direction order = Sort.Direction.ASC;
+
+    @Builder.Default
+    private Boolean owner = true;
```

Comment:
```suggestion
    private boolean owner = true;
```
believe it's easier to understand by renaming it to `byOwner`, to match the repository function name